src/activities/ActivityUsageEntity.js

```diff
@@ -0,0 +1,49 @@
+'use strict';

```

Comment:
I don't _think_ this should be need here, since "strict" mode is default for ES6 modules.